docs/vcpkg-schema-definitions.schema.json

```
@@ -0,0 +1,715 @@
+            },
+            "strip": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            },
+            "transform": {
+              "type": "string",
+              "description": "An SED expression to run on the file names before extracting them."
+            }
+          },
+          "required": [
+            "untar"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "nupkg": {
+              "type": "string",
+              "format": "uri"
+            },
+            "sha256": {
+              "$ref": "#/definitions/sha256"
+            },
+            "sha512": {
+              "$ref": "#/definitions/sha512"
+            },
+            "strip": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            },
+            "transform": {
+              "type": "string",
+              "description": "An SED expression to run on the file names before extracting them."
+            },
+            "lang": {
+              "type": "string",
+              "description": "The locale under which this install should be applied."
+            },
+            "nametag": {
+              "type": "string",
+              "description": "A string to add to the name of the artifact when extracted on disk."
+            }
+          },
+          "required": [
+            "nupkg"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "git": {
+              "type": "string",
+              "description": "The http or ssh remote from which a git repo will be fetched."
+            },
+            "commit": {
+              "type": "string",
+              "description": "The commit SHA or ref that will be checked out."
+            },
+            "options": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "enum": [
+                  "full",
+                  "recurse"
+                ]
+              }
+            },
+            "subdirectory": {
+              "type": "string",
+              "default": ""
+            }
+          },
+          "required": [
+            "git"
+          ],
+          "additionalProperties": false
+        }
+      ]
+    },
+    "git-registry": {
+      "type": "object",
+      "description": "A port registry backed by a git repo.",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "pattern": "git"
+        },
+        "repository": {
+          "type": "string",
+          "description": "The URI or SSH entry to clone from. (as if by `git clone`)"
+        },
+        "baseline": {
+          "description": "The SHA that will be checked out to find default versions for ports from this registry.",
+          "$ref": "#/definitions/sha1"
+        },
+        "ref": {
```

```suggestion
        "reference": {
```

This is actually wrong in the docs.